packages/teleterm/src/ui/TabHost/TabHost.tsx

```diff
@@ -33,8 +33,12 @@ export default function TabHost(props: Props) {
   const documents = serviceDocs.getDocuments();
   const docActive = serviceDocs.getActive();
 
+  const handleTabNew = useCallback(() => {
+    serviceDocs.openNewDocument();
+  }, [serviceDocs]);
+
   // enable keyboard shortcuts
-  useTabShortcuts(ctx);
+  useTabShortcuts(ctx, { openNewTab: handleTabNew });
```

Comment:
do we need to pass `handleNewTab` method? The `serviceDocs.openNewDocument()` could be called from within the `useTabShortcuts` as it has reference to it (via `ctx`)

Reply:
We don't need after the changes, thanks for catching this!